metricbeat/module/vsphere/datastorecluster/_meta/fields.yml

```diff
@@ -25,4 +25,9 @@
     - name: datastore.count
       type: long
       description: >
-        Number of datastores in the Datastore Cluster.
\ No newline at end of file
+        Number of datastores in the Datastore Cluster.
+    - name: triggerd_alarms.*
+      type: object
+      object_type: keyword
+      description: >
+        List of all the triggerd alarms.
```

Comment:
```suggestion
        List of all the triggerd critical (red status) alarms.
```

Reply:
Somthing like above that indicates its the critical alarms only

Reply:
We are collecting all the triggered alarms so the description is correct currently.